Traceback (most recent call last):
  File "udr_ole.py", line 101, in <module>
    main()
  File "udr_ole.py", line 95, in main
    train_test(args, train_env, test_env, w)
  File "udr_ole.py", line 49, in train_test
    custom_callback=myCallback(train_env, width_factor)
  File "/home/chiar/mldl-main/myCall.py", line 6, in __init__
    self.n_steps = n_steps
NameError: name 'n_steps' is not defined